### PR TITLE
fix: fix validation issues

### LINKED ISSUE #90: [Auto] AI Gallery Standard Validation
# AI Gallery Standard Validation: CONFORMING

Please review the warnings below. If your template is already published, it may be subject to removal from a collection.
Head to this link for more information: [template-lifecycle](https://aka.ms/ai-gallery/template-lifecycle)

## Repository Management:
- [x] <b>README.md File</b>.
- [x] <b>LICENSE.md File</b>.
- [x] <b>SECURITY.md File</b>.
- [x] <b>CODE_OF_CONDUCT.md File</b>.
- [x] <b>CONTRIBUTING.md File</b>.
- [x] <b>ISSUE_TEMPLATE.md File</b>.
- [x] <b>Topics on repo contains ['azd-templates', 'ai-azd-templates']</b>.

## Source code structure and conventions:
- [x] <b>azure-dev.yml File</b>.
- [x] <b>azure.yml File</b>.
- [x] <b>infra Folder</b>.
- [x] <b>.devcontainer Folder</b>.

## Functional Requirements:
- [x] <b>azd up</b>.
- [x] <b>azd down</b>.

## Security Requirements:
- [x] <b>Security scan is not performed</b>.

### PR DESCRIPTION
**Pull request by anfibiacreativa**
## Purpose
fixes #90 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by kristenwomack — APPROVED** [review]
LGTM. Note: all the changes bring this template to pass validation rules.

File: README.md [hunk 1]
```diff
@@ -291,18 +292,25 @@ If you do want to re-enable access from your development environment, re-run the
...
+## Guidance
 
-Pricing may vary per region and usage. Exact costs cannot be estimated.
-You may try the [Azure pricing calculator](https://azure.microsoft.com/pricing/calculator/) for the resources below:
+### Region Availability
 
-* Azure Container Apps: Pay-as-you-go tier. Costs are based on vCPU and memory used. [Pricing](https://azure.microsoft.com/pricing/details/container-apps/)
-* Azure OpenAI: Standard tier, GPT, and Ada models. Pricing per 1K tokens used, and at least 1K tokens are used per question. [Pricing](https://azure.microsoft.com/pricing/details/cognitive-services/openai-service/)
-* Azure Monitor: Pay-as-you-go tier. Costs based on data ingested. [Pricing](https://azure.microsoft.com/pricing/details/monitor/)
+This template uses [MODEL 1] and [MODEL 2] which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
```
Comment (anfibiacreativa) [inline README.md:299]: @marlenezw please update this generic section with actual models and region information before merge.

File: README.md [hunk 2]
```diff
@@ -291,18 +292,25 @@ If you do want to re-enable access from your development environment, re-run the
...
 
-## Security Guidelines
+### Costs
 
-This template uses [Managed Identity](https://learn.microsoft.com/entra/identity/managed-identities-azure-resources/overview) built in to eliminate the need for developers to manage these credentials. Applications can use managed identities to obtain Microsoft Entra tokens without having to manage any credentials. We also use Key Vault, specifically for Bing Search, since Managed Identity is currently not implemented for it. Additionally, we have added a [GitHub Action tool](https://github.com/microsoft/security-devops-action) that scans the infrastructure-as-code files and generates a report containing any detected issues. To ensure best practices in your repo we recommend anyone creating solutions based on our templates ensure that the [Github secret scanning](https://docs.github.com/code-security/secret-scanning/about-secret-scanning) setting is enabled in your repos.
+You can estimate the cost of this project's architecture with [Azure's pricing calculator](https://azure.microsoft.com/pricing/calculator/)
+
+* [Azure Product] - [plan type] [link to pricing for product](https://azure.microsoft.com/pricing/)
```
Comment (anfibiacreativa) [inline README.md:306]: @marlenezw please update this generic section with actual product pricing information before merge

File: README.md (hunk 1 above)
Comment (marlenezw) [inline README.md:299]:
```suggestion
This template uses  `gpt-35-turbo-0613`, `gpt-4o` which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
```

**Comment by marlenezw** [comment]
Merging! Thank you! @anfibiacreativa

**Pull request merged**